src/_includes/layouts/post.njk

```diff
@@ -77,6 +77,19 @@
                                 </li>
                             {%- endfor %}
                         </ul>
+                    {%- else %}
+                        <h3 class="mb-3 pt-6 border-t-2">Recommended Articles:</h3>
```

Comment:
I believe a partial would be better, or a paired shortcode? This feels a bit duplicate.